types.js

```diff
@@ -130,6 +131,18 @@ type ExportOptions = {|
     shouldTrim: bool
 |};
 
+export type SJFillImage = {|
```

Comment:
I think this should be called something like `ImageDataReference` instead, but otherwise looks good.